src/client/Microsoft.Identity.Client/Platforms/Android/Broker/BrokerRequest.cs

```diff
@@ -65,7 +65,17 @@ public Uri RedirectUri
             AcquireTokenInteractiveParameters acquireTokenInteractiveParameters)
         {
             BrokerRequest br = FromAuthenticationParameters(authenticationRequestParameters);
-            br.Prompt = acquireTokenInteractiveParameters.Prompt.PromptValue.ToUpperInvariant();
+            var prompt = acquireTokenInteractiveParameters.Prompt.PromptValue;
+
+            if (prompt == Client.Prompt.NoPrompt.PromptValue || prompt == Client.Prompt.NotSpecified.PromptValue)
```

Comment:
Can you apply this fix to iOS as well? If unsure about the value needed, then use the old "select_account"

Reply:
why select account?

---
In reply to: [515239838](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2154#discussion_r515239838) [](ancestors = 515239838)